zipkin-cassandra-core/src/main/java/org/twitter/zipkin/storage/cassandra/CannotReadKeyspaceMetadataException.java

```diff
@@ -0,0 +1,10 @@
+package org.twitter.zipkin.storage.cassandra;
+
+import com.datastax.driver.core.Cluster;
+
+public class CannotReadKeyspaceMetadataException extends RuntimeException {
+
+    public CannotReadKeyspaceMetadataException(String keyspace, Cluster cluster) {
```

Comment:
I'd not expect anyone to recognize this. Please just throw IllegalStateException vs declaring single-use exceptions.